src/pmproxy/src/server.c

```diff
@@ -793,6 +793,7 @@ setup_proxy(uv_timer_t *arg)
     setup_redis_module(proxy);
     setup_http_module(proxy);
     setup_pcp_module(proxy);
+    pmWebTimerRegister(redis_reconnect_worker, proxy);
```

Comment:
Can this call not be made in setup_redis_module?  Its exposing internal details about Redis outside of redis.c ... be nicer if it moved in there, then redis_reconnect_worker could be made static also.

Reply:
Yep, good idea.